vueui/.prettierrc.js

```diff
@@ -1,5 +1,7 @@
 module.exports = {
-    trailingComma: "es5",
-    tabWidth: 2,
-    semi: false,
-  };
\ No newline at end of file
+  trailingComma: 'es5',
+  tabWidth: 2,
+  semi: false,
+  singleQuote: true,
+  printWidth: 120,
```

Comment:
Well, default was 80. I already increased, this is moslty a parameter when long lines have to be broken in to more lines. It has to stay decent to prevent vertical scrooling and long hard to read lines.